resources/gem.rb

```diff
@@ -35,6 +35,7 @@
 property :response_file,          String # Only used to reconfigure
 property :user,                   String
 property :rbenv_version,          String, required: true
+property :root_path,              String, default: lazy { Chef::Rbenv.root_path(node, user) }
```

Comment:
I was wondering why you were using self above, but this is a really smart pattern. I think we should adopt this across the board.